swarm/fuse/swarmfs_test.go

```diff
@@ -351,16 +344,18 @@ func (ta *testAPI) unmountWhenResourceBusy(t *testing.T) {
 
 	actualPath := filepath.Join(testMountDir, "2.txt")
```

Comment:
@egonelbre sorry maybe I should have clarified. I understand the test was passing because this error was previously not being checked, but I was questioning the usefulness of the test if it was passing even though it was invalid. On closer inspection, after changing to `1.txt` it will indeed be useful.